api_types.ts

```diff
@@ -1085,6 +1116,14 @@ export interface UpdateSyncExecutionContext extends ExecutionContext {
    * Information about state of the current sync.
    */
   readonly sync: UpdateSync;
+
+  /**
+   * A service for retrieving the sync state in Coda Brain.
+   * @hidden
+   *
+   * TODO(ebo): unhide this
+   */
+  readonly syncState: SyncStateService;
```

Comment:
Do we actually need this here on updates?

Reply:
I think no